cpg-library/src/main/golang/frontend/handler.go

```diff
@@ -460,6 +524,26 @@ func (this *GoLanguageFrontend) handleReturnStmt(fset *token.FileSet, returnStmt
 	return r
 }
 
+func (this *GoLanguageFrontend) handleIncDecStmt(fset *token.FileSet, incDecStmt *ast.IncDecStmt) *cpg.UnaryOperator {
+	this.LogDebug("Handling decimal increment statement: %+v", *incDecStmt)
+
+	u := cpg.NewUnaryOperator(fset, incDecStmt)
+
+	if incDecStmt.Tok == token.INC {
+		u.SetOperatorCode("++")
```

Comment:
cpg already provides this as a public field of UnararyOperator: OPERATOR_POSTFIX_INCREMENT
shouldn't this be used instead of hardcoding "++"?

Reply:
In theory yes. Accessing this in the go frontend tends to be tedious because I need to go up the stack from Go to Java again. Does not seem to be satisfied for a simple "++" I would say.